event-hubs.api.md

```diff
@@ -0,0 +1,264 @@
+## API Report File for "@azure/event-hubs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/abort-controller';
+import { MessagingError } from '@azure/core-amqp';
+import { RetryOptions } from '@azure/core-amqp';
+import { Span } from '@azure/core-tracing';
+import { SpanContext } from '@azure/core-tracing';
+import { SpanOptions } from '@azure/core-tracing';
+import { TokenCredential } from '@azure/core-amqp';
+import { WebSocketImpl } from 'rhea-promise';
+
+// @public
+export interface Checkpoint {
+    consumerGroup: string;
+    eventHubName: string;
+    fullyQualifiedNamespace: string;
+    offset: number;
+    partitionId: string;
+    sequenceNumber: number;
+}
+
+// @public
+export enum CloseReason {
+    OwnershipLost = "OwnershipLost",
+    Shutdown = "Shutdown"
+}
+
+// @public
+export interface CreateBatchOptions {
+    abortSignal?: AbortSignalLike;
+    maxSizeInBytes?: number;
+    partitionId?: string;
+    partitionKey?: string;
+}
+
+// @public
+export interface EventData {
+    body: any;
+    properties?: {
+        [key: string]: any;
+    };
+}
+
+// @public
+export interface EventDataBatch {
+    // @internal
+    batchMessage: Buffer | undefined;
+    count: number;
+    // @internal
+    _messageSpanContexts: SpanContext[];
+    partitionId?: string;
+    partitionKey?: string;
+    sizeInBytes: number;
+    tryAdd(eventData: EventData, options?: TryAddOptions): boolean;
+}
+
+// @public
+export interface EventHubClientOptions {
+    retryOptions?: RetryOptions;
+    userAgent?: string;
+    webSocketOptions?: WebSocketOptions;
+}
+
+// @public
+export class EventHubConsumerClient {
+    constructor(consumerGroup: string, connectionString: string, options?: EventHubClientOptions);
+    constructor(consumerGroup: string, connectionString: string, eventHubName: string, options?: EventHubClientOptions);
+    constructor(consumerGroup: string, fullyQualifiedNamespace: string, eventHubName: string, credential: TokenCredential, options?: EventHubClientOptions);
+    close(): Promise<void>;
+    static defaultConsumerGroup: string;
```

Comment:
This was a case where we intentionally wanted the "name" in the name 
We were worried that the term `defaultConsumerGroup` would make people think that this is the default consumer group that gets created if they don't create one.

Where as this is supposed to be the default name of the default consumer group. The default consumer group itself could have some other name if the creator of the consumer group chose to do so....